### ISSUE TITLE
1.20.1 World Creation Crashes

### ISSUE THREAD

**Issue by ScorchedPork** [StellarWind22/Fabric-Shield-Lib#164]
Crashes when trying to create a new world with the following error:
[20:35:42] [Worker-Main-11/ERROR]:

[20:35:42] Couldn't load tag c:shields as it is missing following references: fabricshieldlib:fabric_shield (from fabric), fabricshieldlib:fabric_banner_shield (from fabric)
[20:35:43] [Render thread/ERROR]:
 Reported exception thrown!
net.minecraft.class_148: mouseClicked event handler
	at net.minecraft.class_437.method_25412(class_437.java:414) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_1601(class_312.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_22686(class_312.java:169) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.execute(class_1255.java:102) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_22684(class_312.java:169) ~[client-intermediary.jar:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43) ~[lwjgl-glfw-3.3.1.jar:?]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.1.jar:?]
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474) ~[lwjgl-glfw-3.3.1.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:237) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1244) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:802) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[1.20.1-0.14.25.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.14.25.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.25.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.25.jar:?]
Caused by: java.lang.AbstractMethodError: Receiver class com.github.crimsondawn45.fabricshieldlib.initializers.FabricShieldLib$$Lambda$3605/0x00000008015354d8 does not define or inherit an implementation of the resolved method 'abstract net.minecraft.class_3955 create(net.minecraft.class_2960, net.minecraft.class_7710)' of interface net.minecraft.class_1866$class_7711.
	at net.minecraft.class_1866.method_45443(class_1866.java:18) ~[client-intermediary.jar:?]
	at net.minecraft.class_1866.method_8121(class_1866.java:8) ~[client-intermediary.jar:?]
	at net.minecraft.class_1863.method_17720(class_1863.java:135) ~[client-intermediary.jar:?]
	at net.minecraft.class_1863.method_20705(class_1863.java:56) ~[client-intermediary.jar:?]
	at net.minecraft.class_1863.method_18788(class_1863.java:35) ~[client-intermediary.jar:?]
	at net.minecraft.class_4080.method_18790(class_4080.java:13) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.class_4014.method_18365(class_4014.java:69) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18857(class_1255.java:139) ~[client-intermediary.jar:?]
	at net.minecraft.class_525.method_31130(class_525.java:352) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_43462(class_528.java:169) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.<init>(class_528.java:95) ~[client-intermediary.jar:?]
	at net.minecraft.class_526.method_25426(class_526.java:54) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_25423(class_437.java:297) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1507(class_310.java:1080) ~[client-intermediary.jar:?]
	at net.minecraft.class_442.method_19861(class_442.java:159) ~[client-intermediary.jar:?]
	at net.minecraft.class_4185.method_25306(class_4185.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4264.method_25348(class_4264.java:56) ~[client-intermediary.jar:?]
	at net.minecraft.class_339.method_25402(class_339.java:189) ~[client-intermediary.jar:?]
	at net.minecraft.class_4069.method_25402(class_4069.java:38) ~[client-intermediary.jar:?]
	at net.minecraft.class_442.method_25402(class_442.java:299) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_1611(class_312.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_25412(class_437.java:409) ~[client-intermediary.jar:?]
	... 14 more

**Comment by CringeStar** [comment]
Oops sorry! I thought we had fixed this with the release of 1.71., but it seems I forgot to implement the fix into 1.19.4 and 1.20. Look out for a new release on the weekend. Thank you for bringing this up.

**Comment by DroidZed** [comment]
Hello, new problem here when attempting to create a new world (from GitHub Actions build):

```
java.lang.AbstractMethodError: Receiver class com.github.crimsondawn45.fabricshieldlib.initializers.FabricShieldLib$$Lambda$4429/0x0000024681d47758 does not define or inherit an implementation of the resolved method 'abstract net.minecraft.class_3955 create(net.minecraft.class_2960, net.minecraft.class_7710)' of interface net.minecraft.class_1866$class_7711.
	at net.minecraft.class_1866.method_45443(class_1866.java:18)
	at net.minecraft.class_1866.method_8121(class_1866.java:8)
	at net.minecraft.class_1863.method_17720(class_1863.java:135)
	at net.minecraft.class_1863.method_20705(class_1863.java:56)
	at net.minecraft.class_1863.method_18788(class_1863.java:35)
	at net.minecraft.class_4080.method_18790(class_4080.java:13)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
	at net.minecraft.class_4014.method_18365(class_4014.java:69)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)
	at net.minecraft.class_1255.method_18857(class_1255.java:139)
	at net.minecraft.class_310.method_18857(class_310.java:27017)
	at net.minecraft.class_525.method_31130(class_525.java:352)
	at net.minecraft.class_526.method_19944(class_526.java:60)
	at net.minecraft.class_4185.method_25306(class_4185.java:94)
	at net.minecraft.class_4264.method_25348(class_4264.java:56)
	at net.minecraft.class_339.method_25402(class_339.java:189)
	at net.minecraft.class_4069.method_25402(class_4069.java:38)
	at net.minecraft.class_312.method_1611(class_312.java:98)
	at net.minecraft.class_437.method_25412(class_437.java:409)
	at net.minecraft.class_312.method_1601(class_312.java:98)
	at net.minecraft.class_312.method_22686(class_312.java:169)
	at net.minecraft.class_1255.execute(class_1255.java:102)
	at net.minecraft.class_312.method_22684(class_312.java:169)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:237)
	at net.minecraft.class_310.method_1523(class_310.java:1244)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by DroidZed** [comment]
Alrighty, I'll give feedback once it gets released. Thanks.

**Comment by CringeStar** [comment]
Fixed in 1.7.2, will be available on all platforms shortly.

**Comment by DroidZed** [comment]
Another crash happened: [Crash Log](https://pastebin.com/pkUyRksC)
